lib/github.ts

```diff
@@ -148,11 +148,15 @@ async function fetchGitHubRepoContentFromDownloadUrl(
   downloadUrl: string
 ): Promise<void> {
   const resp = await fetchRepoFileByDownloadUrl(downloadUrl);
-  const fileContents =
-    typeof resp.data === 'string'
-      ? resp.data
-      : JSON.stringify(resp.data, null, 2);
-  fs.outputFileSync(dest, fileContents, 'utf8');
+  const contentType = resp.headers['content-type'];
```

Comment:
Could we add some tests to this method while we are here to make sure we preserve the behavior in the future?

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/140/commits/40ac634e64750d92576def4dd61b83471c676656